plugins/telescope/default.nix

```diff
@@ -85,6 +90,11 @@ lib.nixvim.neovim-plugin.mkNeovimPlugin config {
         added to `extraPackages`.
       '';
     };
+
+    iconsPackage = mkPackageOption {
+      name = "nvim-web-devicons";
+      default = pkgs.vimPlugins.nvim-web-devicons;
+    };
```

Comment:
Is the package actually installed anywhere?

Reply:
No, that would only make sense.